pkg/scheduler/algorithm/predicates/csi_volume_predicate.go

```diff
@@ -143,8 +143,9 @@ func (c *CSIMaxVolumeLimitChecker) filterAttachableVolumes(
 		if driverName == "" || volumeHandle == "" {
 			continue
 		}
+		volumeUniqueName := fmt.Sprintf("%s/%s", driverName, volumeHandle)
```

Comment:
I'm unsure if this is case we need to handle.   I believe volumeID must be unique, so we may need to fix mock driver if it's not following this

Reply:
In theory, different CSI drivers may use the same VolumeID for different volumes.